src/models/connectionStore.ts

```diff
@@ -222,25 +261,62 @@ export class ConnectionStore {
     }
 
     /**
-     * Lookup credential store
+     * Lookup credential store with migration support
```

Comment:
describe what the migration is so that future-us can better decide when it's okay to remove migration support.